addons/InstanceDock/InstanceSlot.gd

```diff
@@ -186,13 +187,21 @@ func set_data(data: Dictionary):
 	custom_texture = data.get("custom_texture", "")
 	apply_data()
 
+func set_text_label(isVisible : bool, text : String = """[center]DRAG
```

Comment:
Why is there a text argument if you always use the default?
This method can be removed in favor of `show()` and `hide()`.

Reply:
I was thinking ahead to allow for users to set labels on slots, just didn't have time to fully implement that.

Reply:
Then it should be removed until the full implementation.